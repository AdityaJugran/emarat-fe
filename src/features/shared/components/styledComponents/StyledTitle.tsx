import styled from 'styled-components';
import tw from 'twin.macro';

export default styled.p`
	${tw`
        text-lg
        text-gray-700
        font-semibold
        my-1
    `}
`;
