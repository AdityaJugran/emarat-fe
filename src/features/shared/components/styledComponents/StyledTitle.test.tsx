import StyledTitle from 'features/shared/components/styledComponents/StyledTitle';
import renderer from 'react-test-renderer';

it('Styled Title Snapshot', () => {
	const tree = renderer.create(<StyledTitle />).toJSON();
	expect(tree).toMatchSnapshot();
});
