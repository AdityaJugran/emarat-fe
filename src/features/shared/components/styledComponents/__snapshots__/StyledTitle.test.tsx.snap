// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styled Title Snapshot 1`] = `
<p
  className="sc-bdfBQB bbCVRC"
/>
`;
